Example/Glia Sample Integration/Presenters/ContentViewPresenter.swift

```diff
@@ -0,0 +1,115 @@
+                self?.queue = queue
+                self?.isVideoAvailable = queue?.isMediaTypeSupported(.video) ?? false
+                self?.isAudioAvailable = queue?.isMediaTypeSupported(.audio) ?? false
+                self?.isChatAvailable = queue?.isMediaTypeSupported(.text) ?? false
+            }
+        }
+    }
+
+    // Configure listening of Glia's events
+    private func eventListener() {
+        Glia.sharedInstance.onEvent = { event in
+            switch event {
+            case .started:
+                break
+            case .engagementChanged(_):
+                break
+            case .ended:
+                break
+            case .minimized:
+                break
+            case .maximized:
+                break
+            }
+        }
+    }
+
+    // Start the engagement of a desired `EngagementKind`.
+    func startEngagement(_ kind: EngagementKind) {
+        do {
+            try Glia.sharedInstance.start(
+                kind,
+                configuration: gliaConfiguration.composeConfiguration(),
+                queueID: gliaConfiguration.queueID,
+                visitorContext: nil,
+                theme: theme
+            )
+        } catch {
+            print("Error starting engagement")
+        }
+    }
+
+    // Configure Glia, for Core SDK.
+    private func initializeCoreSDK(completion: @escaping () -> Void) {
+        do {
+            let salemoveConfiguration = try Salemove.Configuration(
+                siteId: gliaConfiguration.siteID,
+                region: gliaConfiguration.environment.salemoveRegion,
+                authorizingMethod: .siteApiKey(id: gliaConfiguration.apiID, secret: gliaConfiguration.apiSecret)
+            )
+
+            Salemove.sharedInstance.configure(
+                with: salemoveConfiguration,
+                completion: completion
+            )
+        } catch {
+            print("Error has happened: \(error.localizedDescription)")
+        }
+    }
+
+    // Customize and apply Theme
+    private func customizeTheme() -> Theme {
+        let themeColors = ThemeColor(
+            primary: UIColor(Colors.gliaPurple),
+            baseLight: .white,
+            baseDark: UIColor(Colors.gray),
+            systemNegative: UIColor.systemPink
+        )
+        let colorStyle = ThemeColorStyle.custom(themeColors)
+
+        theme = Theme(colorStyle: colorStyle, fontStyle: .default, showsPoweredBy: true)
+        theme.chat.backgroundColor = .white
+        theme.minimizedBubble.badge?.backgroundColor = UIColor.systemPink
+        theme.minimizedBubble.badge?.fontColor = .white
+        theme.minimizedBubble.badge?.font = .systemFont(ofSize: 14)
+
+        return theme
+    }
+}
+
+// Check if the desired queue is open and available for a certain media.
+private extension SalemoveSDK.Queue {
+    func isMediaTypeSupported(_ mediaType: MediaType) -> Bool {
+        if state.status == .open && state.media.contains(mediaType) {
```

Comment:
This can also be changed to just `state.status == .open && state.media.contains(mediaType)` since that will return true or false